commit 0db89fa243e5edc5de38c88b369e4c3755c5fb74
Author: Chen Yu <yu.c.chen@intel.com>
Date:   Wed Dec 22 12:31:41 2021 +0800

    ACPI: Introduce Platform Firmware Runtime Update device driver
    
    Introduce the pfr_update driver which can be used for Platform Firmware
    Runtime code injection and driver update [1].
    
    The user is expected to provide the EFI capsule, and pass it to the
    driver by writing the capsule to a device special file. The capsule
    is transferred by the driver to the platform firmware with the help
    of an ACPI _DSM method under the special ACPI Platform Firmware
    Runtime Update device (INTC1080), and the actual firmware update is
    carried out by the low-level Management Mode code in the platform
    firmware.
    
    This change allows certain pieces of the platform firmware to be
    updated on the fly while the system is running (runtime) without the
    need to restart it, which is key in the cases when the system needs to
    be available 100% of the time and it cannot afford the downtime related
    to restarting it, or when the work carried out by the system is
    particularly important, so it cannot be interrupted, and it is not
    practical to wait until it is complete.
    
    Link: https://uefi.org/sites/default/files/resources/Intel_MM_OS_Interface_Spec_Rev100.pdf # [1]
    Tested-by: Hongyu Ning <hongyu.ning@intel.com>
    Signed-off-by: Chen Yu <yu.c.chen@intel.com>
    [ rjw: Subject and changelog edits ]
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

diff --git a/Documentation/userspace-api/ioctl/ioctl-number.rst b/Documentation/userspace-api/ioctl/ioctl-number.rst
index cfe6cccf0f44..687efcf245c1 100644
--- a/Documentation/userspace-api/ioctl/ioctl-number.rst
+++ b/Documentation/userspace-api/ioctl/ioctl-number.rst
@@ -367,6 +367,7 @@ Code  Seq#    Include File                                           Comments
                                                                      <mailto:aherrman@de.ibm.com>
 0xE5  00-3F  linux/fuse.h
 0xEC  00-01  drivers/platform/chrome/cros_ec_dev.h                   ChromeOS EC driver
+0xEE  00-09  uapi/linux/pfrut.h                                      Platform Firmware Runtime Update and Telemetry
 0xF3  00-3F  drivers/usb/misc/sisusbvga/sisusb.h                     sisfb (in development)
                                                                      <mailto:thomas@winischhofer.net>
 0xF6  all                                                            LTTng Linux Trace Toolkit Next Generation